raspimouse/src/raspimouse_component.cpp

```diff
@@ -436,11 +436,14 @@ void Raspimouse::velocity_command(const geometry_msgs::msg::Twist::SharedPtr msg
   linear_velocity_ = msg->linear.x;
   angular_velocity_ = msg->angular.z;
 
-  auto forward_hz = 80000 * linear_velocity_ / (9 * M_PI);
-  auto rotation_hz = 400 * angular_velocity_ / M_PI;
+  const auto WHEEL_DIAMETER = get_parameter(WHEEL_DIAMETER_PARAM).get_value<double>();
+  const auto WHEEL_TREAD = get_parameter(WHEEL_TREAD_PARAM).get_value<double>();
+
+  const double vel_left  = (linear_velocity_ - angular_velocity_ * WHEEL_TREAD / 2.0)/(WHEEL_DIAMETER/2);
+  const double vel_right = (linear_velocity_ + angular_velocity_ * WHEEL_TREAD / 2.0)/(WHEEL_DIAMETER/2);
```

Comment:
```suggestion
  const double vel_left = (linear_velocity_ - angular_velocity_ * WHEEL_TREAD / 2.0) /
    (WHEEL_DIAMETER / 2);
  const double vel_right = (linear_velocity_ + angular_velocity_ * WHEEL_TREAD / 2.0) /
    (WHEEL_DIAMETER / 2);
```

ament_lint_autoを使っているのでそれにマッチするようにスタイルの修正をお願いします

`ament_uncrustify`コマンドで再フォーマットできます

参考：https://autowarefoundation.gitlab.io/autoware.auto/AutowareAuto/contributor-guidelines.html#contributors-guidelines-formatting